### PR TITLE
add cpanato as official maintainer

### PR DESCRIPTION
**Pull request by Issif**
Signed-off-by: Issif <issif_github@gadz.org>

**What this PR does / why we need it**:

I proposed in last community call to officially accept @cpanato as an official maintainer of `falcosidekick`. Everybody agrees, so this PR adds him into our beloved famiglia 😉 

**Which issue(s) this PR fixes**:

N/A

Fixes #

N/A

**Special notes for your reviewer**:

Thank you @cpanato for your help with this project 🙏

### CODE REVIEW

**Review by KeisukeYamashita — APPROVED** [review]
nice 🚀

**Comment by KeisukeYamashita** [comment]
@Issif Is there a procedure for me to become such a member too?

**Comment by nibalizer** [comment]
:+1:

**Review by leogr — APPROVED** [review]
I couldn't wait to approve this PR!!! :smiley: 

Congrats @cpanato :partying_face:

**Comment by danpopSD** [comment]
Congratulations Carlos!

**Comment by danpopSD** [comment]
> @Issif Is there a procedure for me to become such a member too?

we would love to have you as well. we will propose this on the next community call!  @KeisukeYamashita

**Comment by KeisukeYamashita** [comment]
> > @Issif Is there a procedure for me to become such a member too?
> 
> 
> 
> we would love to have you as well. we will propose this on the next community call!  @KeisukeYamashita 


@danpopSD Thank you very much. I've made several contributions so far, not just to falcosidekick, and I'd like to contribute more.
I look forward to hearing from you.

File: OWNERS [hunk 1]
```diff
@@ -3,8 +3,10 @@ approvers:
   - leodido
   - nibalizer
   - leogr
+  - cpanato
 reviewers:
   - Issif
   - leodido
   - nibalizer
   - leogr
+  - cpanato
```
Comment (cpanato) [inline OWNERS:12]: need to add a new line

also can order alphabetically?

**Comment by leodido** [comment]
@cpanato you should have an invitation in your inbox 🥇 

image

**Review by leodido — APPROVED** [review]

File: OWNERS (hunk 1 above)
Comment (Issif) [reply OWNERS:12]: @cpanato Will do so in next change 😉 

@KeisukeYamashita I'll ask in next community, if everybody agrees like I do, I'll create the PR 👍, don't hesitate to come more often on Slack for chatting with everybody and discussing about changes.

**Pull request merged**